File: aat-android/src/main/java/ch/bailu/aat/preferences/map/SolidDem3EnableDownload.java
package ch.bailu.aat.preferences.map;

import ch.bailu.aat_lib.preferences.SolidBoolean;
import ch.bailu.aat_lib.preferences.StorageInterface;
import ch.bailu.aat_lib.resources.ToDo;

public class SolidDem3EnableDownload extends SolidBoolean {
    public SolidDem3EnableDownload(StorageInterface storageInterface) {
        super(storageInterface, SolidDem3EnableDownload.class.getSimpleName());
    }

    @Override
    public String getLabel() {
        return ToDo.translate("Enable automatic download");
    }
}

File: aat-lib/src/main/java/ch/bailu/aat_lib/preferences/map/SolidDem3EnableDownload.kt
package ch.bailu.aat_lib.preferences.map

import ch.bailu.aat_lib.preferences.SolidBoolean
import ch.bailu.aat_lib.preferences.StorageInterface
import ch.bailu.aat_lib.resources.Res


class SolidDem3EnableDownload(storage: StorageInterface) : SolidBoolean(
    storage, SolidDem3EnableDownload::class.java.simpleName
) {
    
    override fun getLabel(): String {
        return Res.str().p_dem_auto_download()
    }
}
